fhir-install/README-DB2.md

```diff
@@ -0,0 +1,120 @@
+# IBM FHIR Server Install - Using docker container db2 as data store
+
+Running fhir server integration tests using docker db2.
+
+## Prerequisites
+
+- [Docker]
+
+## Build
+- Access the fhir-install/docker directory;
+- Copy Dockerfile-fhirdb2 and create-database.sh to a newly created directory;
+- Rename Dockerfile-fhirdb2 to Dockerfile;
+- Using Docker Terminal, access the new directory and run:
+
+```sh
+docker build -t fhirserverdb2 . --squash
+```
+## Run
+ 
+Once the image is built, start it with:
+
+```sh
+docker run -itd --name fhirdb2 --privileged=true -p 50000:50000 -e LICENSE=accept -e DB2INST1_PASSWORD=change-password -v /db2 -rm fhirserverdb2
+```
+if docker fails to find the image by name, then please use the image id instead of name for this above command.
```

Comment:
add a line between the code block and text

Reply:
good point, will change

Reply:
done

Reply:
I think the volume should be there. 

```
docker run -itd --name mydb2 --privileged=true -p 50000:50000 -e LICENSE=accept -e DB2INST1_PASSWORD=<choose an instance password> -e DBNAME=testdb -v <db storage dir>:/database ibmcom/db2
```

Else the volume is destroyed... maybe mention it in the document?

Reply:
correct, done!